content/en/integrations/guide/aws-integration-troubleshooting.md

```diff
@@ -59,10 +59,14 @@ When installing the Agent on an AWS host, you might see duplicated hosts on the
 
 ### EC2 metadata with IMDS v2
 
-In your [Agent configuration][5], if the parameter `ec2_prefer_imdsv2`, is set to `true` (defaults to `false`), the Agent requests EC2 metadata using Instance Metadata Service Version 2 (IMDSv2), which offers additional security for accessing metadata. In some situations, additional configuration may be required in AWS, for example: using a containerized Agent on a plain EC2 instance. See the [Transition to using Instance Metadata Service Version 2][6] documentation for details.
+EC2's [IMDSv2][8] can sometimes lead to problems gathering EC2 instance hostnames, where the Agent uses the `os` hostname provider instead of `aws`, as seen in the output of `agent status`.
 
 In containerized environments the problem might be that you have locked down the EC2 metadata endpoint, by way of assigning IAM roles/credentials to pods running in the Kubernetes cluster. `Kube2IAM` and `kiam` are common tools used to do this. To solve this, update your `Kube2IAM` or `kiam` configuration to allow access to this endpoint.
 
+The AWS API supports disabling IMDSv1, which the Agent uses by default. If this is the case, set the parameter `ec2_prefer_imdsv2` to `true` (defaults to `false`) in your [Agent configuration][5]. See the [Transition to using Instance Metadata Service Version 2][6] documentation for details.
+
+IMDSv2, in its default configuration, refuses connections with an IP hop count greater than one, that is, connections that have passed through an IP gateway. This can cause problems when the Agent is running in a container with a network other than the host's network, as the runtime forwards the container's traffic through a virtual IP gateway. This is common in ECS deployments. To remedy this, increase the maximum hop count to at least `2`.
```

Comment:
Should we explain how to increase the count to 2 ? Maybe link to some documentation ?

Also is there any down side of this ? I'm guessing this was done to prevent external access to IMDSv2, should we sync with security ?